src/Indexer.js

```diff
@@ -246,6 +247,13 @@ export default class Indexer {
     return dates.length > 0 ? dates[0] : undefined
   }
 
+  buildRDFTypes(indexObject, json) {
+    const rdfTypes = json['@graph']
+      .filter((item) => item['@type'] && !Array.isArray(item['@type']))
```

Comment:
This enforces that the `@type` is a single URI and not an array.

Reply:
This doesn't seem like a great way of identifying the correct `@type`.